From 886f7f4004e55f4eb3d61c15ec10d64878fc1dbd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Vr=C3=A1til?= <dvratil@kde.org>
Date: Fri, 12 Jul 2019 12:19:25 +0200
Subject: Fix build due to missing QVector include

Summary:
I get 'implicit instantiation of undefined template 'QVector<QString>'
when building against Qt 5.12.4 without this change.

Test Plan: Builds against Qt 5.12.4

Reviewers: dfaure, davidedmundson

Reviewed By: davidedmundson

Subscribers: kde-frameworks-devel

Tags: #frameworks

Differential Revision: https://phabricator.kde.org/D22414
---
 src/kpackage/private/versionparser.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/kpackage/private/versionparser.cpp b/src/kpackage/private/versionparser.cpp
index 71dcd08..2411d2a 100644
--- a/src/kpackage/private/versionparser.cpp
+++ b/src/kpackage/private/versionparser.cpp
@@ -21,6 +21,8 @@
 
 #include "package.h"
 
+#include <QVector>
+
 
 namespace KPackage
 {
-- 
cgit v1.1
